### PR TITLE
Adding capacity parameter in ebs volumes list

### PR DESCRIPTION
**Pull request by pratik97**
Signed-off-by: Pratik Gandhi <pratikgandhi1997@gmail.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** https://github.com/openebs/openebs/issues/1980#issuecomment-427289861

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by pratik97** [comment]
@ashishranjan738 can you guide me how to pass codecov/patch test?

**Comment by sonasingh46** [comment]
@pratik97 
Thank you.
Can you please put the issue link in PR description.

**Comment by pratik97** [comment]
@sonasingh46 Added the link.

File: cmd/mayactl/app/command/volumes_list.go [hunk 1]
```diff
@@ -63,15 +63,15 @@ func (c *CmdVolumeOptions) RunVolumesList(cmd *cobra.Command) error {
 	}
 
 	out := make([]string, len(cvols.Items)+2)
-	out[0] = "Namespace|Name|Status|Type"
-	out[1] = "---------|----|------|----"
+	out[0] = "Namespace|Name|Status|Type|Capacity"
+	out[1] = "---------|----|------|----|--------"
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volumes_list.go]: @pratik97 Thanks for contribution...
Can you please paste the output of `volume list` command which will show capacity of a openebs volume..that will be helpful.

**Pull request merged**